CalcualtorLogic.js

```diff
@@ -107,8 +107,6 @@ class Calculator {
         const currentResultAndclickedButton = this.currentResult + event.target.dataset.number;
         this.currentResult = currentResultAndclickedButton;
         this.updateResultElement();
```

Comment:
what is the reason of calling `        this.updateResultElement();` here?

Reply:
I do not know